docs/querying/filters.md

```diff
@@ -30,6 +30,10 @@ sidebar_label: "Filters"
 
 A filter is a JSON object indicating which rows of data should be included in the computation for a query. It’s essentially the equivalent of the WHERE clause in SQL. Apache Druid supports the following types of filters.
 
+**Note**
+
+Filters are commonly applied on dimensions, but can be applied on metrics as well.
```

Comment:
Should this link to `aggregations.md#filtered-aggregator`?

Reply:
In my head, i was thinking filtering on aggregates, like filtering on metrics in a groupby. Sure, adding a link to those would be useful here. 
Separately, does anyone know if we started supporting filtering metrics since a particular release version which should be specified here ?